docs/generated/oc_by_example_content.adoc

```diff
@@ -2003,20 +2003,39 @@ Change the number of pods in a deployment
 
 
 == oc secrets add
-Add secrets to a ServiceAccount
+Link secrets to a ServiceAccount (deprecated)
 
 ====
 
 [options="nowrap"]
 ----
-  // To use your secret inside of a pod or as a push, pull, or source secret for a build, you must add a 'mount' secret to your service account like this:
-  oc secrets add serviceaccount/sa-name secrets/secret-name secrets/another-secret-name
+  # To use your secret inside of a pod or as a push, pull, or source secret for a build, you must link a 'mount' secret to your service account like this:
```

Comment:
Sorry, I don't entirely understand. How would they use this secret otherwise? Can you suggest an alternative phrasing?